src/multidim/arithmetic.jl

```diff
@@ -24,3 +24,24 @@ wrap(v) = v
 Base.broadcast(f, X::IntervalBox) = wrap(f.(X.v))
 Base.broadcast(f, X::IntervalBox, Y::IntervalBox) = wrap(f.(X.v, Y.v))
 Base.broadcast(f, X::IntervalBox, y) = wrap(f.(X.v, y))
+
+for op in (:+, :-, :∩, :∪, :⊆, :isinterior, :dot, :setdiff)
+    @eval $(op)(a::SVector{N, Interval{T}}, b::IntervalBox{N, T}) where {N, T<:Real} = $(op)(IntervalBox(a), b)
```

Comment:
You should now be able to write this as
```
$(op)(a::SVector, b::IntervalBox)
```
(and similarly for the other lines below)
and it should just work if it's an `SVector` of `Real`s.

Reply:
Okay so let the methods for IntervalBox determine whether they can be added instead of explicitly specifying the types here?